serve_mux_test.go

```diff
@@ -70,3 +75,46 @@ func BenchmarkMuxMatch(b *testing.B) {
 	b.Run("lowercase", bench("_dns._udp.example.com."))
 	b.Run("uppercase", bench("_DNS._UDP.EXAMPLE.COM."))
 }
+
+func BenchmarkMuxMatchConcurrent(b *testing.B) {
```

Comment:
I think this test should just using `testing`s builtin parallel benchmark support.

Reply:
:+1:, changed it